### DISCUSSION TITLE
Tart installer pkg

### DISCUSSION THREAD

**Discussion by wegotoeleven** [cirruslabs/tart#655] [General]
Hey! A while back, Tart releases also included a pkg installer. Are there plans to bring this back at all?

Thanks!

**Comment by luckman212** [comment]
Came here to ask the same. Is there any way to install Tart on a machine **without** Homebrew?

**Reply by fkorotkov** [reply]
You can manually install the binary:

https://github.com/cirruslabs/tart/blob/9bb71a4051e4d377dad2993f4f3ee6635dfec132/docs/quick-start.md?plain=1#L15-L25

Does that work for you?

**Reply by luckman212** [reply]
Hmm, actually yes it does. I originally tried it and because the icon looked like this I thought it had failed:

<img width="243" alt="image" src="https://github.com/cirruslabs/tart/assets/1992842/3997d311-1c28-4983-9da2-676faeb0e3be">

I guess I was expecting to see this nice colorful Tart icon:

<img width="256" src="https://github.com/cirruslabs/tart/assets/1992842/8c310fee-3825-4461-bc55-9ae3809ce768">

**Comment by kevinmcox** [comment]
I came to ask the same thing. Are there any plans to start including the installer package in releases again?
